default/scripting/policies/CONFEDERATION.focs.txt

```diff
@@ -18,7 +18,7 @@ Policy
                 Not ResourceSupplyConnected empire = LocalCandidate.Owner condition = Source
             ]
             effects = SetTargetInfluence value = Value
-                        + (NamedRealLookup name = "SUPPLY_DISCONNECTED_INFLUENCE_FLAT")
+                        + (NamedReal name = "PLC_CONFEDERATION_SUPPLY_DISCONNECTED_INFLUENCE_FLAT" value = 1)
```

Comment:
This was a typo from me (still WIP), I have a SUPPLY_DISCONNECTED_INFLUENCE_FLAT and SUPPLY_DISCONNECTED_INFLUENCE_MALUS.

There are two places where this is (should) be referenced, here and in default/scripting/species/common/influence.macros. The point is Confederation nullifies the influence malus from supply-disconnection, so if at some point someone changes supply-disconnection malus then Confederation has to be updated. Hence the reference value in named_values.macros.

Reply:
not sure what you propose - I guess using SUPPLY_DISCONNECTED_INFLUENCE_MALUS ?

Reply:
it is not helpful if the FLAT is defined name-in-the-middle and the MALUS in named_values

or one could use a macro for the flat value and use that in both NamedReal definitions

Reply:
I'm not sure what are "name-in-the-middle".
The idea is to have SUPPLY_DISCONNECTED_INFLUENCE_FLAT declared in named.values and use NamedRealLookup in CONFEDERATION and influence.macros. That's the current state in my next PR (not yet finished and pushed).

Reply:
> or one could use a macro for the flat value and use that in both NamedReal definitions

That's better (than what I had in mind).

Reply:
i will implement that then